xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/AdminWikisRightsSheet.xml

```diff
@@ -39,7 +39,7 @@
   <content>{{velocity}}
 ### Sheet used to generically display the XWikiPreferences object fields in the administration sheets.
 {{html}}
-  &lt;form method="post" action="$xwiki.getURL($currentDoc, 'saveandcontinue')" class="xform"&gt;
+  &lt;form method="post" class="xform"&gt;
```

Comment:
I'm not sure about this change. Can you give more details about the error you got? It looks as if you're changing the place where the form is submitted and I'm surprised it still works.

Reply:
> CSRF validation failed when saving

It looks like the form is missing the CSRF token. Does it work if you add the hidden input with the form token? (the JavaScript code is probably adding it dynamically when submitting the form).

Reply:
Indeed. I should have looked better the stacktrace :)
I'll add the hidden input then and put back the form URL. Do you think I should remove the dynamic add of the token in the JS then?

Reply:
> Do you think I should remove the dynamic add of the token in the JS then?

Sure, if it's not too complex.